src/command-line/upgrade.js

```diff
@@ -0,0 +1,57 @@
+"use strict";
+
+const colors = require("chalk");
+const program = require("commander");
+const Helper = require("../helper");
+const Utils = require("./utils");
+
+program
+	.command("upgrade")
+	.option("-p, --package [name]", "Update only the specified package")
```

Comment:
`Upgrade`

Also, could it be possible to make the command `thelounge upgrade [package]` (or ``thelounge upgrade [...packages]`)? If `package` is specified, only upgrades this package(s), and if not upgrades them all.
I don't know to which extent commander would let you do that.

Reply:
I tried but there was no way to make it an optional argument so it would not allow the command to be executed without the package name.